src/lib/features/feature-toggle/converters/feature-toggle-row-converter.ts

```diff
@@ -123,7 +125,10 @@ export class FeatureToggleRowConverter {
         feature.variants = row.variants || [];
         feature.project = row.project;
 
-        if (this.isUnseenStrategyRow(feature, row) && !row.strategy_disabled) {
+        if (
+            this.isUnseenStrategyRow(feature, row) &&
+            (includeDisabledStrategies ? true : !row.strategy_disabled)
+        ) {
```

Comment:
I don't think this is the correct place to put this. createBaseFeature should only care about what is common between the two API endpoints. Instead you should take this check and move into the respective methods: `buildFeatureToggleListFromRows` and `buildPlaygroundFeaturesFromRows` and have each method have different checks for which strategies they allow

Reply:
Ok makes sense

Reply:
Turns out that both of those functions are only used for the Admin API - for which we (everywhere) return the disabled strategies